src/foundation/layout.css

```diff
@@ -0,0 +1,5 @@
+:root {
+  --page-padding-x: 6rem;
+  --page-padding-y: 3rem;
+  --main-content-percentage-width: calc(2 / 3);
+}
```

Comment:
These variables are required in order to have the search box the same width as the main content.